code/game/objects/items.dm

```diff
@@ -632,3 +640,34 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 		M.dirty++
 
 /obj/item/proc/on_mob_death(mob/living/L, gibbed)
+
+/obj/item/proc/update_force_string()
+	if(force)
+		switch(force)
+			if(1 to 3)
+				force_string = "very low"
+			if(4 to 6)
+				force_string = "low"
+			if(7 to 9)

```

Comment:
this needs to be `if(6 to 9)`, so that it includes forces of 6.5. Same for all the others, it has to be inclusive.